geoips/plugins/yaml/sectors/static/goes17.yaml

```diff
@@ -15,7 +15,7 @@ spec:
   projection:
     a: 6371228.0
     lat_0: 0
-    lon_0: -137.2
+    lon_0: 137.2
```

Comment:
@jsolbrig Is there a reason this changed?

Reply:
Ugh, that again? Why does that keep coming back? No, this should be negative! Fixing.